deploy/helm/internal-charts/storage-api/values.yaml

```diff
@@ -9,3 +9,8 @@ resources:
 minio:
   host: "minio"
   port: 9000
+  nameOverride: "magda-minio"
+  fullnameOverride: "magda-minio"
+  certsPath: "/etc/minio/certs/"
```

Comment:
We probably don't need set `certsPath ` & `configPathmc ` as the value we set here is same as the default value?